### ISSUE TITLE
Improve ways for community to give feedback

### ISSUE THREAD

**Issue by StanczakDominik** [PlasmaPy/PlasmaPy#624]
Turns out Github may not be the best way to have people give feedback. Who knew?

* Possibilities: 
    * a suggestion box available from the README file
        * Use google forms
            * Manually repost to google group for now
    * maybe a discussion forum?
        * we already have https://groups.google.com/forum/#!forum/plasmapy
            * advertise it RIGHT ON THE FRONT PAGE
                * README
                * plasmapy.org
                * Docs.plasmapy.org
                * A button saying FORUM RIGHT HERE or something :)
        * https://www.discourse.org/
            * https://blog.discourse.org/2018/11/free-hosting-for-open-source-v2/
    * https://github.com/PlasmaPy/PlasmaPy/graphs/traffic
    * A survey of the whole plasma community using google forms
    * Should have a “feedback” heading or “communication” heading in main README file

I'll handle that within a week and a half if nobody catches it before I do. Feel free, though!
